<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SerialCfgGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>串口配置区</string>
    </property>
    <widget class="QLabel" name="StopBitLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QLabel" name="PortLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QLabel" name="BaudLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="BaudComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ParityBitLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="DataBitComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="StopBitComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="PortComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="DataBitLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ParityBitComboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ScanRangeGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>扫描范围设置</string>
    </property>
    <widget class="QLineEdit" name="XRangeLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="YRangeLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PointNumLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="XLabel">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>53</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X方向</string>
     </property>
    </widget>
    <widget class="QLabel" name="YLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>83</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y方向</string>
     </property>
    </widget>
    <widget class="QLabel" name="PointNumLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>113</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>显示点数</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveScanRangePushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ComGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>801</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>通信区</string>
    </property>
    <widget class="QLabel" name="SendDataLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>103</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发送数据</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SendDatalineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>102</y>
       <width>501</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="RecDataLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>52</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>接收数据</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="HEXRecCheckBox">
     <property name="geometry">
      <rect>
       <x>595</x>
       <y>49</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AsciiRecCheckBox">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>51</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>ASCII</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RecClearPushButton">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="HEXSendCheckBox">
     <property name="geometry">
      <rect>
       <x>597</x>
       <y>103</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AsciiSendCheckBox">
     <property name="geometry">
      <rect>
       <x>651</x>
       <y>104</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>ASCII</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendPushButton">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>102</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="RecDataTextBrowser">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>501</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="WaveDisplayGroupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>731</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>波形显示区</string>
    </property>
    <widget class="QCustomPlot" name="WaveWidget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>671</width>
       <height>391</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="StartPushButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IndicateLabel">
    <property name="geometry">
     <rect>
      <x>891</x>
      <y>54</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/images/Resouce/image.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>15</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CECPort 毫米波雷达调试工具</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="AboutMenu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="AboutMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">Qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
